src/components/Popover/Popover.scss

```diff
@@ -7,7 +7,7 @@ $content-max-width: rem(400px);
 
 .Popover {
   max-width: calc(100vw - #{2 * spacing()});
-  margin: $visible-portion-of-arrow spacing(tight) spacing();
+  margin: $visible-portion-of-arrow 0 0;
```

Comment:
I'm not sure why the popover needed margin on the horizontal axis as it is positioned absolutely.